client/packages/coldchain/src/Equipment/DetailView/Tabs/Summary.tsx

```diff
@@ -246,12 +246,15 @@ export const Summary = ({ draft, onChange, locations }: SummaryProps) => {
             />
           </Row>
         </Section>
-        {draft.properties.length === 0 ? null : (
+        {draft.parsedProperties ? null : (
           <Section heading={t('label.catalogue-properties')}>
-            {draft.properties.map(property => (
-              <Row key={property.id} label={property.name}>
+            {Object.keys(draft.parsedProperties).map(key => (
+              <Row
+                key={key}
+                label={draft.parsedProperties[key]?.toString() ?? ''}
```

Comment:
Is this ok? I figure it probalby is?

Reply:
Will be updated in Next PR anyway...

Reply:
I feel like parsed communicates the intention better.
When I refactor catalogueProperties I'll revisit...